src/pages/resources/hackathon-resources.mdx

```diff
@@ -41,12 +31,27 @@ Always refer back to the [getting help](/resources/getting-help) page for the la
 - [Hackathon Discord](https://discord.gg/h3AvTHj)
 - [Developer Discord](https://discord.gg/2YHSAey)
 
-## Tutorials
+## Documentation and tutorials
+
+If you are new to Smart Contracts, read the [Getting Started Guide](/getting-started/conceptual-overview/).
 
-Be sure to check the documentation in the links above for in depth-descriptions of everything.
+### Service documentation
 
-New? See the [Getting Started Guide](/getting-started/conceptual-overview/).
+- [Data Feeds](/data-feeds)
+  - [Using Data Feeds](/data-feeds/using-data-feeds)
+  - [Rate and Volatility Feeds](/data-feeds/rates-feeds)
+- [VRF](/vrf/v2/introduction)
+- [Automation](/chainlink-automation/introduction/)
+- [Chainlink Functions](/chainlink-functions)
+- [Any API](/any-api/introduction)
+- [Chainlink Nodes](/chainlink-nodes)
+  - [Running a Chainlink Node](/chainlink-nodes/v1/running-a-chainlink-node)
 
+### Tutorials
+
+In addition to the examples in the Documentation, you can also find several tutorials.
+
+- [Learning Resources](/getting-started/other-tutorials)
 - [Video Tutorials](https://www.youtube.com/playlist?list=PLVP9aGDn-X0QwJVbQvuKr-zrh2_DV5M6J)
```

Comment:
Should we also add Chaindev in here? Alot of the content there is tutorial 'how to' type stuff 

https://www.youtube.com/@Chain_Dev

Reply:
Added